src/ChunkedStreamDecoder.php

```diff
@@ -81,7 +81,9 @@ protected function iterateBuffer()
             $this->emit('data', array(
                 substr($this->buffer, 0, $chunkLength),
                 $this,
-                $this->chunkedExtension
+                [
+                    'chunkedExtension' => $this->chunkedExtension,
```

Comment:
It should probably be `chunkExtension` not `chunkedExtension`as i propesed in my comment.